.github/workflows/upstream-dev-ci.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
         id: status
         run: |
           set -euo pipefail
-          python -m pytest -rf | tee output-${{ matrix.python-version }}-log || (
+          python -m pytest --timeout=7200 -rf | tee output-${{ matrix.python-version }}-log || (
```

Comment:
```suggestion
          python -m pytest --timeout=5400 -rf | tee output-${{ matrix.python-version }}-log || (
```

Reply:
thanks, I had missed that the limit already was 2h

Reply:
Yes, the title was wrong...